services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/model/BatchWriteItemEnhancedRequest.java

```diff
@@ -92,10 +137,53 @@ private static List<WriteBatch> getListIfExist(List<WriteBatch> writeBatches) {
     @NotThreadSafe
     public static final class Builder {
         private List<WriteBatch> writeBatches;
+        private String returnConsumedCapacity;
+        private String returnItemCollectionMetrics;
 
         private Builder() {
         }
 
+        /**
+         * Whether to return the capacity consumed by this operation.
+         *
+         * @see Builder#returnConsumedCapacity(ReturnConsumedCapacity)
+         */
+        public Builder returnConsumedCapacity(ReturnConsumedCapacity returnConsumedCapacity) {
+            this.returnConsumedCapacity = returnConsumedCapacity == null ? null : returnConsumedCapacity.toString();
+            return this;
+        }
+
+        /**
+         * Whether to return the capacity consumed by this operation.
+         *
+         * @see Builder#returnConsumedCapacity(String)
+         */
+        public Builder returnConsumedCapacity(String returnConsumedCapacity) {
+            this.returnConsumedCapacity = returnConsumedCapacity;
+            return this;
+        }
+
+        /**
+         * Whether to return the item collection metrics.
```

Comment:
Nit: this method does not return a boolean value though. Suggesting
```
Sets the item collection metrics.
```

Reply:
Fixed